src/__init__.py

```diff
@@ -45,6 +48,15 @@ def initCSS(self, *args):
     #_flag[style*="rgb(119, 170, 255)"]::after, #_flag[style*="--flag4-fg"]::after {
       content: %s;
     }
+    #_flag[style*="rgb(240, 151, 228)"]::after, #_flag[style*="--flag5-fg"]::after {
```

Comment:
These are the exact RGB values that anki uses for their new flags, FYI

Reply:
Recently anki changed classname to `--flag--%d`.

line should be changed to
```css
    #_flag[style*="rgb(255, 102, 102)"]::after, #_flag[style*="--flag1-fg"]::after, #_flag[style*="--flag-1"]::after {
```

Reply:
Convo was marked as resolved. Was there anything to be done for this? @phu54321

Reply:
Unresolving :)